occlum/run_one_tests_on_occlum.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+TEST_CASE=$*
+TEST_CASE_DIR=/opt/occlum/gvisor_syscall_tests
+BLOCKLIST_DIR=../blocklist
+FILTER=""
+
+get_blocklist_subtests(){
+	if [ -f $BLOCKLIST_DIR/$TEST_CASE ]; then
+		FILTER=$(cat $BLOCKLIST_DIR/$TEST_CASE | tr '\n' ':')
+		return 0
+	else
+		FILTER=""
+		return 1
+	fi
+}
+
+if [ -e /$TEST_CASE_DIR/$TEST_CASE ];then
+	continue
```

Comment:
continue shouldn't be used here. Please refine the if statements.

Reply:
Changes have been submitted